style.css

```diff
@@ -145,6 +145,81 @@ section.mobile-menu a {
   grid-template-columns: 1fr;
 }
 
+/* popup window mobile  */
+
+.modal-mobile {
+  position: fixed;
+  top: 0;
+  left: 0;
+  margin-top: 18px;
+  margin-left: 16px;
+  margin-right: 16px;
+  background-color: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 1.6rem;
+  width: auto;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  gap: 1.2rem;
+  padding: 1.6rem;
+}
```

Comment:
- [ ] You have done very great to implement the popup modal. However, we need to ensure that the popup follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0-1&t=5l8w9aWjJfBV2v6d-0) (e.g., font, colors, images, text, margins). Therefore I suggest that we change the `height` of the modal so that all the contents of the modal can be displayed and also not overlap with each other.
- [ ] In addition, try as much as possible to ensure the modal is responsive at all the desktop and mobile version sizes and also have a `blur` effect to match the Figma design☺️ 

Desktop 😎 
![image](https://user-images.githubusercontent.com/99394374/223688900-37d25b7f-5cf2-456b-8926-fce335d3d46a.png)

Mobile version 😎 
![image](https://user-images.githubusercontent.com/99394374/223689012-7a739934-33ed-4da7-b277-395121963a6f.png)

Figma design mobile 😎 
![image](https://user-images.githubusercontent.com/99394374/223678979-698173a7-4370-474d-8430-b905859aba30.png)

Figma desktop
![image](https://user-images.githubusercontent.com/99394374/223679134-ac37feb2-0bc3-4f76-8d2f-d95c486dbf96.png)